proto/api/v1/operations.proto

```diff
@@ -41,6 +41,8 @@ message ListOperationsRequest {
   // Default value is `created_at desc`
   // NOTE: On http protocol, this operates as a query parameter.
   string order_by = 2;
+  // Optional parameter for enabling filter by operation status
+  string status = 3;
```

Comment:
AFAIK the status is not a required parameter as `page` and `per_page`, what do you think to add the `optional` parameter? This link could help you https://developers.google.com/protocol-buffers/docs/proto#optional